### ISSUE TITLE
ENB Doesn't seem to load within FiveM

### ISSUE THREAD

**Issue by HunterHoch** [NarutoUA/gta5_fsr#65]
ENB Doesn't seem to load within FiveM. I followed the instructions exactly but can't seem to get ENB to actually work

**Comment by Kromi77** [comment]
That's true the menu of ENB doesn't show up when you install fsr mode :(

**Comment by NarutoUA** [comment]
There is similar proxy configuration in enb's ini file. Try to rename fsr to d3d11_fsr.dll, enb to d3d11.dll and change enb config similar to fsr. I hope you understand the idea

**Comment by Yankwerks** [comment]
It does not work by using enb's proxy function still.

**Comment by azure-hu** [comment]
> There is similar proxy configuration in enb's ini file. Try to rename fsr to d3d11_fsr.dll, enb to d3d11.dll and change enb config similar to fsr. I hope you understand the idea

Triied that method with FiveM. I Use ReShade 5.2.2 too, and ENB version is 0.4.7.1, that comes with QuantV's 2022 April package.

In enbloca.ini these values were set:
[PROXY]
EnableProxyLibrary=true
InitProxyFunctions=true
ProxyLibrary=d3d11_fsr.dll

In gta5_frs.ini: 
[PROXY]
EnableProxyLibrary=false
InitProxyFunctions=false
ProxyLibrary=

Same files has debug enabled:
[DEBUG]
PrintDebug=true

Graphics are set to 2560x1440 resolution in advanced settings, and 1920x1080 as render resolution. ENB menu works, and in DebugView, when FiveM starts, it shows: 
[13288] DirectX11 DLL's found.
[13288] GTA5_FSR: PrintDebug is 1
[13288] GTA5_FSR: Reading config file...
[13288] GTA5_FSR: UseFidelityFxSuperResolution is 1
[13288] GTA5_FSR: AutoSharpness is 1
[13288] GTA5_FSR: Sharpness: 0.500000
[13288] GTA5_FSR: EnableProxyLibrary is 0
but when I start story mode, no other GTA5_FSR debug lines are shown, that were displayed before, when ENB was renamed and set to GTA5_FSR.ini but not worked.

**Comment by Councilord22** [comment]
Anyone found a solution for this yet?

**Comment by Kv0th** [comment]
> There is similar proxy configuration in enb's ini file. Try to rename fsr to d3d11_fsr.dll, enb to d3d11.dll and change enb config similar to fsr. I hope you understand the idea

I can't make it work in fivem with ENB (QuantV), 

- [I do have enb d3d11 as `d3d11.dll` and fsr d3d11 as `d3d11_fsr.dll`](https://i.imgur.com/YS4MfAp.png)
- [I also have `gta5_fsr.ini` with false proxy status](https://i.imgur.com/ahTuKn8.png) 
- and [`enblocal.ini` with true proxy status](https://i.imgur.com/tIVkEAV.png)

Anyone knows how to help me?

My ENB menu works, but fsr don't. If I remove ENB it also works.

(I'm not using reshade, but I would love to know if I can)

**Comment by sl1ki0** [comment]
Does anyone found a solution?

**Comment by sl1ki0** [comment]
I don't have fsr files at all, but it's not working

**Comment by sl1ki0** [comment]
I found solution! Put your ENB files in fivem directiory in plugins folder
